include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -1047,6 +979,7 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUImage** const _begin,
             blitRegion.dstSubresource.mipLevel = i;
             blitRegion.dstOffsets[1] = { mipWidth, mipHeight, mipDepth };
 
+            // TODO: Remove the requirement that the transfer queue has graphics caps,
             cmdbuf_transfer->blitImage(gpuimg, asset::IImage::EL_TRANSFER_SRC_OPTIMAL, gpuimg,
                 asset::IImage::EL_TRANSFER_DST_OPTIMAL, 1u, &blitRegion, asset::ISampler::ETF_LINEAR);
```

Comment:
can you use @achalpandeyy's Compute Blitter ?

Reply:
Yea but not now, I planned it as part of Asset Converter 2.0 work